soapui-testserver-api/src/main/java/com/smartbear/ready/recipe/TestStepNames.java

```diff
@@ -12,4 +14,20 @@
     public static final String DELAY_TYPE = "Delay";
     public static final String PROPERTIES_TYPE = "Properties";
     public static final String SOAP_MOCK_RESPONSE_TYPE = "SOAPMockResponse";
+
+    public static String createUniqueName(WsdlTestCase testCase, String testStepName) {
+        if (testCase.getTestStepByName(testStepName) == null) {
+            return testStepName;
+        }
+
+        String orgName = testStepName;
+        int cnt = 1;
+
+        while (testCase.getTestStepByName(testStepName) != null) {
+            testStepName = orgName + " " + String.valueOf(cnt);
+            cnt++;
+        }
+
```

Comment:
There's some unnecessary newlines in this commit, I think. This is one of them. Think about which ones you really want to keep.